diff --git a/simulators/ethereum/consensus/forks.go b/simulators/ethereum/consensus/forks.go
index ed74c718..6bb3c6b4 100644
--- a/simulators/ethereum/consensus/forks.go
+++ b/simulators/ethereum/consensus/forks.go
@@ -306,10 +306,10 @@ var envForks = map[string]map[string]int{
 		"HIVE_TERMINAL_TOTAL_DIFFICULTY": 0,
 		"HIVE_SHANGHAI_TIMESTAMP":        15000,
 	},
-	"Cancun": {
-		"HIVE_FORK_HOMESTEAD":            0,
-		"HIVE_FORK_TANGERINE":            0,
-		"HIVE_FORK_SPURIOUS":             0,
+		"Cancun": {
+			"HIVE_FORK_HOMESTEAD":            0,
+			"HIVE_FORK_TANGERINE":            0,
+			"HIVE_FORK_SPURIOUS":             0,
 		"HIVE_FORK_BYZANTIUM":            0,
 		"HIVE_FORK_CONSTANTINOPLE":       0,
 		"HIVE_FORK_PETERSBURG":           0,
@@ -318,12 +318,28 @@ var envForks = map[string]map[string]int{
 		"HIVE_FORK_LONDON":               0,
 		"HIVE_FORK_MERGE":                0,
 		"HIVE_TERMINAL_TOTAL_DIFFICULTY": 0,
-		"HIVE_SHANGHAI_TIMESTAMP":        0,
-		"HIVE_CANCUN_TIMESTAMP":          0,
-	},
-	"ShanghaiToCancunAtTime15k": {
-		"HIVE_FORK_HOMESTEAD":            0,
-		"HIVE_FORK_TANGERINE":            0,
+			"HIVE_SHANGHAI_TIMESTAMP":        0,
+			"HIVE_CANCUN_TIMESTAMP":          0,
+		},
+		"Prague": {
+			"HIVE_FORK_HOMESTEAD":            0,
+			"HIVE_FORK_TANGERINE":            0,
+			"HIVE_FORK_SPURIOUS":             0,
+			"HIVE_FORK_BYZANTIUM":            0,
+			"HIVE_FORK_CONSTANTINOPLE":       0,
+			"HIVE_FORK_PETERSBURG":           0,
+			"HIVE_FORK_ISTANBUL":             0,
+			"HIVE_FORK_BERLIN":               0,
+			"HIVE_FORK_LONDON":               0,
+			"HIVE_FORK_MERGE":                0,
+			"HIVE_TERMINAL_TOTAL_DIFFICULTY": 0,
+			"HIVE_SHANGHAI_TIMESTAMP":        0,
+			"HIVE_CANCUN_TIMESTAMP":          0,
+			"HIVE_PRAGUE_TIMESTAMP":          0,
+		},
+		"ShanghaiToCancunAtTime15k": {
+			"HIVE_FORK_HOMESTEAD":            0,
+			"HIVE_FORK_TANGERINE":            0,
 		"HIVE_FORK_SPURIOUS":             0,
 		"HIVE_FORK_BYZANTIUM":            0,
 		"HIVE_FORK_CONSTANTINOPLE":       0,
